src/concepts/pipeline/filtering.md

```diff
@@ -0,0 +1,105 @@
+# Filtering Data
+
+## Overview
+
+Data is filtered out of production streams at almost every stage of the pipeline.
+The following outlines each stage and both the data currently filtered and the
+data that could be filtered. This should help answer two classes of questions:
+
+1. Did my data get filtered out?
+2. We've uncovered spurious data being ingested, how should we handle that?
+
+*Note*: JSON Schema filtering is our primary method of filtering out bad data. That should be used before any other methods of dropping data from the pipeline.
+
+## Stages
+
+Where - Which stage of the pipeline this filtering occurs in
+
+What - What happens to the data when filtered here
+
+When - Which situations this filtering should be, and is, used in
+
+How - What kind of data can be filtered at this stage
+
+### Edge filtering
+
+Where: Filtered by nginx, [currently we use it to filter out non-v4 pings](https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-ingestion/k8s/charts/data-ingestion/templates/filter-configmap.yaml#L14-L41)
+([to be removed, but capability to remain](https://bugzilla.mozilla.org/show_bug.cgi?id=1678497)).
+
+What: Drops data entirely from the pipeline; there will be no traces of it downstream from the edge server.
+
+When: Only to be used in extreme situations (e.g. PII exposure). We also use it for dropping [too-large messages](https://github.com/mozilla/gcp-ingestion/blob/master/docs/architecture/overview.md#limits) and [headers](https://github.com/mozilla/gcp-ingestion/blob/master/ingestion-edge/ingestion_edge/util.py#L95).
+
+How: Can be used to filter by URI namespaces, apps, etc.; but not anything in the payload. 
```

Comment:
Specifically technically, this means it will filter on "HTTP stuff" (headers or url pieces) only?

Reply:
It does - I will clarify